app/src/main/java/edu/gatech/gtorg/gitmad/threads/fragment/ThreadDetailsFragment.java

```diff
@@ -110,14 +114,17 @@ public void onClick(View view) {
             }
         });
 
+        /* TODO 12 (Advanced)
+         *  Create a new View.OnClickListener() for sendButton by using an anonymous inner class
```

Comment:
nit: downloadButton, not send button here. It may also be helpful to tell them to set the downloadButton's onClickListener to an anonymous instance of View.OnClickListener().